Makefile

```diff
@@ -322,17 +322,17 @@ apply_sklearn_initmerge:
 
 cleanjenkins: mrproper cleancpp cleanc cleanpy xgboost_clean py3nvml_clean
 
-buildjenkins: update_submodule cpp c py
+buildjenkins: update_submodule cpp c
 
-installjenkins: pyinstall
+buildpy: py pyinstall
 
 # for nccl cuda8 build
-fullinstalljenkins: cleanjenkins alldeps_private buildjenkins installjenkins
+buildjenkins: cleanjenkins alldeps_private buildjenkins
```

Comment:
In general, i would recommend to keep standard execution paths, if there is a separated execution path for building at jenkins, then it can easily miss problems which are visible by developers.

Reply:
@mmalohlava yes, the Makefile is due for some cleanup now that we don't use pydatatables and probably can be simplified